scripts/rebalance-queue-masters

```diff
@@ -115,6 +116,22 @@ set_temp_queue_policies()
             node_master_count[$2]="$1"
             IFS=$'\n'
         done
+
+        # Find polices that have mirroring enabled.
+        IFS=$'\n'
+        for tmp in $(rabbitmqctl -p openstack list_policies | grep 'ha-mode')
```

Comment:
We can't accept changes specific to OpenStack.

Reply:
Sorry, that was an oversight on my part. I have fixed it with a generic `$vhost`. Would that work?